doc/develop/modules.rst

```diff
@@ -569,6 +569,36 @@ Build files located in a ``MODULE_EXT_ROOT`` can be described as:
 This allows control of the build inclusion to be described externally to the
 Zephyr module.
 
+SBOM integration
+====================
+
+The module description file :file:`zephyr/module.yml` can be used to improve the
+generated SBOM and vulnerability monitoring.
+
+The version and the CPE associated to the module can be provided using
+``version`` and ``cpe`` fields:
+
+.. code-block:: yaml
+
+   version: <module-version>
```

Comment:
Let's move the `version` out of `SBOM integration`.

`version` has value on its own, and should be considered first class citizen in the module yaml.
Just like `name` is independent: https://docs.zephyrproject.org/latest/develop/modules.html#module-name